x-pack/plugins/apm/public/components/shared/transaction_action_menu/transaction_action_menu.tsx

```diff
@@ -69,8 +70,22 @@ export function TransactionActionMenu({ transaction, isLoading }: Props) {
   const { isProfilingPluginInitialized, profilingLocators } =
     useProfilingPlugin();
 
+  const [isCreateEditFlyoutOpen, setIsCreateEditFlyoutOpen] = useState(false);
+
+  const openCustomLinkFlyout = function (){
```

Comment:
```suggestion
 function openCustomLinkFlyout() {
```

For consistency in code, can you please change it to use the function syntax